__tests__/components/Notification.test.tsx

```diff
@@ -57,35 +59,12 @@ describe('Notification component', () => {
     it('should display a children component when defined', () => {
         const DummyComp = () => <div>Connect To Wallet</div>;
         render(
-            <Notification title="Wallet is disconnected">
+            <ENotification title="Wallet is disconnected">
                 <DummyComp />
-            </Notification>
+            </ENotification>
         );
 
         expect(screen.getByText('Wallet is disconnected')).toBeInTheDocument();
         expect(screen.getByText('Connect To Wallet')).toBeInTheDocument();
     });
-
-    it('should display default accent colour and different accents based on status', () => {
```

Comment:
This test was tightly coupled with the alert styles and was flaky between chakra v2 and v3. Also, it was just testing if chakra was correctly styling the Alert component based on the passed `status`. We can trust chakra to do its thing without us testing it. That's why I removed it.